docs(i18n): contributing guidelines in polish

mathias3 opened the pull request:
<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

mathias3 committed:
creating folder with docs in polish

mathias3 committed:
Merge pull request #1 from mathias3/mathias3-polish-folder

creating folder with docs in polish

mathias3 committed:
start contributing.md in polish

mathias3 committed:
added rest of article

mathias3 committed:
rapair markdown links

mathias3 committed:
first draft of devops.md

mathias3 committed:
delete first draft od devops.md

kassjja committed:
fix translation in Contributing.md

kassjja committed:
Merge pull request #1 from kassjja/fix-translation-in-Contributing.md

fix translation in Contributing.md

mathias3 committed:
Merge pull request #2 from kassjja/master

fix translation in Contributing .md

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+Ściśle egzekwujemy nasz [„Kodeks postępowania”](https://www.freecodecamp.org/code-of-conduct). Poświęć chwilę, aby go przeczytać. Ma tylko 196 słów.
+
+Szczęsliwego rozwijania kodu open source!
```
"Szczęśliwego"

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+## Oto kilka zabawnych sposobów, w które możesz pomóc
+
+Możesz przyczynić się do jednego z tych:
```
"[...] do jednego z tych rzeczy:"

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+- [Kategoria "Współtwórcy" na naszym forum publicznym](https://www.freecodecamp.org/forum/c/contributors)
+- [Nasz publiczny współpracownik - chat room na Gitterze](https://gitter.im/FreeCodeCamp/Contributors)
+
+Jesteśmy podekscytowani, że pomożemy Ci wziąć udział we wszystkich tematach, nad którymi chciałbyś pracować. Zachęcamy do zadawania nam pytań na tematy pokrewne, a my chętnie udzielimy wyjaśnień. Upewnij się, że szukasz swojego zapytania przed wysłaniem nowego. Bądź uprzejmy i cierpliwy. Nasza społeczność wolontariuszy i moderatorów jest zawsze w pobliżu, aby poprowadzić Cię przez Twoje pytania.
```
"Jesteśmy podekscytowani, by pomóc Ci ze wzięciu udziału"

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+Możesz przyczynić się do jednego z tych:
+
+ 
+1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#przyczyniaj się do tej otwartej źródłowej bazy danych). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.. Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiać błędy na platformie edukacyjnej.
```
```suggestion
1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```

Qondor reviewed (changes requested):
Corrected few typos and grammar mistakes, but there may be more of them.

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+Cześć. 👋
+
+funkcjonowanie freeCodeCamp.org jest możliwe dzięki tysiącom życzliwych wolontariuszy takich jak Ty. Jesteśmy wdzięczni za Twój wkład i cieszymy się, że możemy Cię powitać na pokładzie.
```
```suggestion
Funkcjonowanie freeCodeCamp.org jest możliwe dzięki tysiącom życzliwych wolontariuszy takich jak Ty. Jesteśmy wdzięczni za Twój wkład i cieszymy się, że możemy powitać Cię na pokładzie.
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+Szczęśliwego rozwijania kodu open source!
+
+## Oto kilka zabawnych sposobów, w które możesz pomóc
```
```suggestion
## Oto kilka fajnych sposobów, w które możesz pomóc
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+Możesz przyczynić się do jednego z tych rzeczy:
+
+ 
+1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```
```suggestion
1. [Rozwijaj tę otwartoźródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```
This is the proper translation of "open-source" https://sjp.pwn.pl/poradnia/haslo/otwartozrodlowy;10114.html

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+ 
+1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
+
+2. Pomoc dla kamperów na naszym [forum publicznym](https://www.freecodecamp.org/forum/). [Odpowiedz na pytania dotyczące kodowania](https://www.freecodecamp.org/forum/?max_posts=1) lub [podaj im informacje zwrotne na temat ich projektów kodowania](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1).
```
```suggestion
2. Pomoc dla kamperów na naszym [publicznym forum](https://www.freecodecamp.org/forum/). [Odpowiedz na pytania dotyczące kodowania](https://www.freecodecamp.org/forum/?max_posts=1) lub [podaj im informacje zwrotne na temat ich projektów kodowania](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1).
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+2. Pomoc dla kamperów na naszym [forum publicznym](https://www.freecodecamp.org/forum/). [Odpowiedz na pytania dotyczące kodowania](https://www.freecodecamp.org/forum/?max_posts=1) lub [podaj im informacje zwrotne na temat ich projektów kodowania](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1).
+
+3. Pomóż nam dodawać napisy (podpisy zamknięte) do naszych [Filmów na kanale YouTube](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ/videos).
```
```suggestion
3. Pomóż nam dodawać napisy do naszych [filmów na kanale YouTube](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ/videos).
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+3. Pomóż nam dodawać napisy (podpisy zamknięte) do naszych [Filmów na kanale YouTube](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ/videos).
+
+## Przyczyniaj się do tej otwartej źródłowej bazy danych.
```
```suggestion
## Przyczyniaj się do tej otwartoźródłowej bazy danych.
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+##### Twórz, aktualizuj i naprawiaj błędy w naszych wyzwaniach związanych z kodowaniem.
+
+Wszystkie nasze wyzwania związane z kodowaniem są obsługiwane przez społeczność, dostarczającą specjalistyczną wiedzę od wolontariuszy takich jak Ty.
```
```suggestion
Wszystkie nasze wyzwania związane z kodowaniem są obsługiwane przez społeczność dostarczającą specjalistyczną wiedzę od wolontariuszy takich jak Ty.
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+- Mamy serwer API oparty na Node.js.
+- Zestaw aplikacji klienckich opartych na React.
+- Skrypt, którego używamy do oceny naszych projektów front-end.
+rozwijanie naszego stacku wymaga trochę zrozumienia API, ES6 Syntax, i chęci nauki.
```
```suggestion

Rozwijanie naszego stacku wymaga trochę zrozumienia API, ES6 Syntax, i chęci nauki.
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+Zasadniczo oczekujemy podstawowej znajomości niektórych z wyżej wymienionych technologii, narzędzi i bibliotek. Biorąc to pod uwagę, nie jesteście zobowiązani do bycia ekspertem w tym zakresie, aby móc wnieść swój wkład.
+
+**Jeśli chcesz pomóc nam udoskonalić naszą bazę danych, możesz albo [skonfigurować freeCodeCamp lokalnie](/docs/how-to-setup-freecamp-locally.md) albo użyć Gitpod, darmowego środowiska online dev.**
```
```suggestion
**Jeśli chcesz pomóc nam udoskonalić naszą bazę danych, możesz albo [skonfigurować freeCodeCamp lokalnie](/docs/how-to-setup-freecamp-locally.md) albo użyć Gitpod, darmowego internetowego środowiska deweloperskiego.**
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+[![Otwórz w Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/freeCodeCamp/freeCodeCamp)
+
+(Uruchamia gotowe do kodowania środowisko dev dla freecodecamp w przeglądarce.)
```
```suggestion
(Uruchamia gotowe do kodowania środowisko deweloperskie dla freecodecamp w przeglądarce.)
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+### Jak mogę zgłosić błąd, którego nie ma na pokładzie?
+
+Jeśli uważasz, że znalazłeś błąd, przeczytaj najpierw artykuł ["Help I've Found a Bug"](https://forum.freecodecamp.org/t/how-to-report-a-bug/19543) i postępuj zgodnie z jego instrukcjami.
```
```suggestion
Jeśli uważasz, że znalazłeś błąd, przeczytaj najpierw artykuł ["Pomocy znalazłem błąd"](https://forum.freecodecamp.org/t/how-to-report-a-bug/19543) i postępuj zgodnie z jego instrukcjami.
```

Other links have polish names too, so I suppose this one should have as well?

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+Jeśli jesteś pewien, że jest to nowy błąd, stwórz nowy problem z GitHub. Upewnij się, że dołączysz jak najwięcej informacji, abyśmy mogli odtworzyć błąd. Mamy predefiniowany szablon problemu, który pomoże Ci w jego rozwiązaniu.
+
+Proszę zwrócić uwagę, że wszelkie problemy, które szukają pomocy przy kodowaniu w przypadku wyzwania, zostaną zamknięte. Śledzenie spraw jest ściśle związane z zagadnieniami związanymi z bazą danych i dyskusjami. W razie wątpliwości, przed sporządzeniem raportu należy [szukać pomocy na forum](https://www.freecodecamp.org/forum).
```
```suggestion
Zwróć uwagę, że wszelkie problemy, które szukają pomocy przy kodowaniu w przypadku wyzwania, zostaną zamknięte. Śledzenie spraw jest ściśle związane z zagadnieniami związanymi z bazą danych i dyskusjami. W razie wątpliwości, przed sporządzeniem raportu należy [szukać pomocy na forum](https://www.freecodecamp.org/forum).
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+##### Jak mogę zgłosić problem bezpieczeństwa?
+
+Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego, wyślij e-mail na adres `security@freecodecamp.org` i natychmiast się temu przyjrzymy.
```
```suggestion
Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego wyślij e-mail na adres `security@freecodecamp.org` i natychmiast się temu przyjrzymy.
```

Qondor replied:
```suggestion
Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego wyślij e-mail na adres `security@freecodecamp.org` a natychmiast się temu przyjrzymy.
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+Nasi moderatorzy społeczności [triage](https://en.wikipedia.org/wiki/Software_bug#Bug_management) zgłaszają problemy i wnioski w oparciu o ich priorytet, dotkliwość i inne czynniki. Możesz [znaleźć kompletny słowniczek ich znaczeń tutaj](https://github.com/freecodecamp/freecodecamp/labels).
+
+Powinieneś przejść przez [**`help wanted`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`first timers welcome`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 
```
```suggestion
Powinieneś przejść przez [**`potrzebna pomoc`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`nowicjusze mile widziani`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 
```

Qondor replied:
```suggestion
Powinieneś przejść przez [**`potrzebna pomoc`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`nowicjusze mile widziani`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+Powinieneś przejść przez [**`help wanted`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`first timers welcome`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 
+
+Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem pracy nad nimi. Jeśli te kwestie nie są jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.
```
```suggestion
Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem nad nimi pracy. Jeśli w kwestiach tych nie jest jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.
```

Qondor replied:
```suggestion
Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem nad nimi pracy. Jeśli w kwestiach tych nie jest jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+#### Znalazłem literówkę, czy powinienem zgłosić problem, zanim złożę prośbę o ciągnięcie?
+
+W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Kwestie są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.
```
```suggestion
W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Problemy są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.
```

Qondor replied:
```suggestion
W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Problemy są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+##### Jak mogę dostać problem przypisany do mnie?
+
+Zazwyczaj nie przydziela się problemów nikomu innemu niż długoletni współpracownicy, aby uniknąć ambitnego nie pojawienia się na zajęciach. Zamiast tego postępujemy zgodnie z poniższą polityką, aby być sprawiedliwym dla wszystkich:
```
```suggestion
Zazwyczaj nie przydziela się problemów nikomu innemu niż długoletnim współpracownikom, aby uniknąć ambitnego nie dostarczenia wyników. Zamiast tego postępujemy zgodnie z poniższą polityką, aby być sprawiedliwym dla wszystkich:
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+##### Jestem nowy w GitHubie i Open Source, od czego powinienem zacząć?
+
+Przeczytaj nasz [How to Contribute to Open Source Guide](https://github.com/freeCodeCamp/how-to-contribute-to-open-source).
```
```suggestion
Przeczytaj nasz [Poradnik jak wnieść wkład do Open Source](https://github.com/freeCodeCamp/how-to-contribute-to-open-source).
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+Przeczytaj nasz [How to Contribute to Open Source Guide](https://github.com/freeCodeCamp/how-to-contribute-to-open-source).
+
+##### Utknęłam na czymś, co nie jest zawarte w tej dokumentacji. Jak mogę uzyskać pomoc?
```
```suggestion
##### Utknąłem na czymś, co nie jest zawarte w tej dokumentacji. Jak mogę uzyskać pomoc?
```

Qondor commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+Możesz poprosić o pomoc:
+
+- [Kategoria "Współtwórcy" na naszym forum publicznym](https://www.freecodecamp.org/forum/c/contributors)
+- [Nasz publiczny współpracownik - chat room na Gitterze](https://gitter.im/FreeCodeCamp/Contributors)
```
```suggestion
- [Nasz publiczny chat room dla współpracowników na Gitterze](https://gitter.im/FreeCodeCamp/Contributors)
```

Qondor reviewed (changes requested):
I have corrected grammar and syntax errors. Seem okay now.

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

przyjmuje sugestie - moim pierwszym pomysłem było, żeby nie tłumaczyć takich tekstów , żeby łatwo było zobaczyc , że to ten sam link i prowadzi do tego samego so we wersji anglojezycznej. widziałem zę w innych tłumaczeniach prowadzi to do lekkiego zamieszania UX

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

słusznie

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+Możesz przyczynić się do jednego z tych rzeczy:
+
+ 
+1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```
wymagana poprawka do tej poprawki- masz tu literówkę

Qondor replied:
Corrected too.

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+ - Podążał pan za listą kontrolną? 
+ - Czy podałeś nazwę swojej listy kontrolnej w sposób znaczący?
+
+Nie potrzebujesz żadnego zezwolenia dla spraw, które są oznaczone jako "pomoc poszukiwana" lub "pierwsze powitanie czasowe", jak wyjaśniono wcześniej. Postępuj zgodnie z wytycznymi i otwórz prośbę o pomoc.
```
literówka "potrzeban pomoc"

Qondor replied:
Great catch, thanks.

mathias3 committed:
add contact table

mathias3 commented on docs/i18n-languages/polish/CONTRIBUTING.md:
```diff
@@ -0,0 +1,107 @@
+
+Jesteśmy podekscytowani, by pomóc Ci we wzięciu udziału we wszystkich tematach, nad którymi chciałbyś pracować. Zachęcamy do zadawania nam pytań na tematy pokrewne, a my chętnie udzielimy wyjaśnień. Upewnij się, że szukasz swojego zapytania przed wysłaniem nowego. Bądź uprzejmy i cierpliwy. Nasza społeczność wolontariuszy i moderatorów jest zawsze w pobliżu, aby poprowadzić Cię przez Twoje pytani ...
+
+Jeśli masz wątpliwości, możesz skontaktować się z naszym zespołem programistów platformy, aby Ci w tym pomóc:
```
done

RandellDawson commented:
@mathias3 You have an earlier PR (#37401) which creates a CONTRIBUTING.md file.  Please either close this PR and make the same changes to #37401 or close #37401 and add its changes to this PR.  You also have another PR (#37384) which creates a CONTRIBUTING.md file.  You should only have a single PR which makes changes to the CONTRIBUTING.md file.  There can be other files on the same PR, but no multiple PRs for the same file.  Merging these would cause a substantial merge conflict.

Thank you.

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 committed:
Update docs/i18n-languages/polish/CONTRIBUTING.md

Co-Authored-By: Cezary Waś <44169282+Qondor@users.noreply.github.com>

mathias3 commented:
> @mathias3 You have an earlier PR (#37401) which creates a CONTRIBUTING.md file. Please either close this PR and make the same changes to #37401 or close #37401 and add its changes to this PR. You also have another PR (#37384) which creates a CONTRIBUTING.md file. You should only have a single PR which makes changes to the CONTRIBUTING.md file. There can be other files on the same PR, but no multiple PRs for the same file. Merging these would cause a substantial merge conflict.
> 
> Thank you.
Done. I deleted contributing.md from both (#37401) and(#37384)

RandellDawson commented:
@mathias3 I should have chosen my words more carefully.  You can't just delete the file from the PR, because that would delete the file from the repo.  Instead you need to revert the changes to the file on the PR, so it is as if you never made any changes to it.

mathias3 committed:
Create devops.md

mathias3 committed:
Create how-to-open-a-pull-request.md

mathias3 committed:
translate fix ins devops.md

mathias3 committed:
create how to work on...

mathias3 commented:
> @mathias3 You have an earlier PR (#37401) which creates a CONTRIBUTING.md file. Please either close this PR and make the same changes to #37401 or close #37401 and add its changes to this PR. You also have another PR (#37384) which creates a CONTRIBUTING.md file. You should only have a single PR which makes changes to the CONTRIBUTING.md file. There can be other files on the same PR, but no multiple PRs for the same file. Merging these would cause a substantial merge conflict.
> 
> Thank you.

ok, i closed all others pull request to avoid merge conflict. All files are in this pr and they were checked by Qondor and few others for  spell check

RandellDawson commented:
@raisedadead Do we have any reviewers for this one?

raisedadead commented:
I do not believe we have anyone, but I am going ahead and merging this for the hard work that has been put in by @mathias3 :)
